docs/configuration/index-config.md

```diff
@@ -410,6 +412,53 @@ field_mappings:
     type: text
 ```
 
+#### concatenate
+
+Quickwit supports mapping the content of multiple fields to a single one. This can be more efficient at query time than
+searching through dozens of `default_search_fields`. It also allow querying inside a json field without knowing the path
+to the field being searched.
+
+```yaml
+name: my_default_field
+type: concatenate
+concatenated_fields:
+  - text # things inside text, tokenized with the `default` tokenizer
+  - resource.author # all fields in resource.author, assuming resource is a `json` field
```

Comment:
```suggestion
  - resource.author # all fields in resource.author, assuming resource is a `json` field
```
Does it work on an object type too?

Reply:
i got a bit confused when redoing the documentation. This is for object field, not json fields.  Currently we wouldn't visit inside author if it was an object. The code is architectured so it wouldn't be hard to add, and would likely be done at the same time as adding support for wildcard names (like `param_*`)